src/components/Dropdown/Dropdown.vue

```diff
@@ -85,12 +110,15 @@
           >
             <dropdown-item
               :id="`${id}-${flattenedOptions.indexOf(item)}`"
-              :ref="activeIndex === flattenedOptions.indexOf(item) ? 'activeOption' : null"
+              :ref="(el) => setOptionRef(el, item)"
```

Comment:
See thread here on what the problem was with setting this `ref` like this inside a `v-for`: https://lob.slack.com/archives/C01LGSJ3DFZ/p1652454604921389.  I think using a function is the proper pattern to follow now when doing something like this where there should only be 1 item rendered in the `v-for` that has a given ref.